packages/gator-permissions-snap/src/permissions/nativeTokenStream/content.tsx

```diff
@@ -34,6 +41,22 @@ export async function createConfirmationContent({
 
   return (
     <Box>
+      <Section>
+        {renderRules({
+          rules: [initialAmountRule, maxAmountRule],
+          context,
+          metadata,
+        })}
+        <Box>
+          <Divider />
+        </Box>
```

Comment:
The `nativeTokenStream` wraps the Divider in an extra Box:
 ```tsx
<Box>
    <Divider />
  </Box>
```
  While `erc20TokenStream` uses it directly:
 ```tsx
<Divider />
```
  Both `erc20TokenPeriodic` and `nativeTokenPeriodic` also use it directly without wrapping. This inconsistency may cause different visual rendering between permission types. recommend to remove the Box wrapper in `nativeTokenStream/content.tsx`